eng/Versions.props

```diff
@@ -67,6 +67,7 @@
     <MicrosoftDotNetRemoteExecutorVersion>7.0.0-beta.22316.2</MicrosoftDotNetRemoteExecutorVersion>
     <cdbsosversion>10.0.18362</cdbsosversion>
     <NewtonSoftJsonVersion>13.0.1</NewtonSoftJsonVersion>
+    <MicrosoftSourceBuildIntermediatearcadePackageVersion>9.0.0-beta.24076.5</MicrosoftSourceBuildIntermediatearcadePackageVersion>
```

Comment:
IMO, these intermediate properties should be cleaned up as they are never referenced.  This adds bloat to the file and a bunch of revision edits each time the dependency is updated for no benefit.

Reply:
Removed in https://github.com/dotnet/diagnostics/pull/4487/commits/ce02767d1db397f44b8f15f005349fa8db9b742e